### ISSUE TITLE
up the Contrast ratio for accessibility 

### ISSUE THREAD

**Issue by CampbellDocherty** [fac19/week5-FKAM#41]

**Comment by freemvmt** [comment]
Check out [contrast checker](https://contrastchecker.com/) for a quick check on this . There's tools like this all over the internet. I also had to [convert](https://convertacolor.com/) your hsl to hex to use it. The good news is your black text on that background is WCAG AAA compliant (that's peak accessibility) - it's just the orange title that's killing your game ;)
